crates/e2e/macro/src/codegen.rs

```diff
@@ -174,47 +174,45 @@ impl InkE2ETest {
 
 /// Builds the contract at `manifest_path`, returns the path to the contract
 /// bundle build artifact.
-fn build_contract(manifest_path: &str) -> String {
-    use std::process::{
-        Command,
-        Stdio,
+fn build_contract(path_to_cargo_toml: &str) -> String {
+    use contract_build::{
+        BuildArtifacts,
+        BuildMode,
+        ExecuteArgs,
+        ManifestPath,
+        Verbosity,
+    };
+
+    let manifest_path =
+        ManifestPath::new(path_to_cargo_toml).expect("Invalid manifest path");
+    let args = ExecuteArgs {
+        manifest_path,
+        verbosity: Verbosity::Default,
+        build_mode: BuildMode::Debug,
+        network: Default::default(),
+        build_artifact: BuildArtifacts::All,
+        unstable_flags: Default::default(),
+        optimization_passes: Default::default(),
+        keep_debug_symbols: false,
+        lint: false,
+        output_type: Default::default(),
```

Comment:
This was `OutputType::Json` before, I guess changing this is fine now since we don't manually need to parse the build output, right?

Reply:
Correct :+1:

Reply:
Based on this  observation I've made all options explicit instead of relying on `Default::default()` which could yield some invisible breaking changes, e.g. I recently changed `OptimizationPasses` to an `Option`.